webpack/node-fetch.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import path from "path";
+
+export default {
+  entry: "./node_modules/node-fetch/src/index.js",
+  output: {
+    path: path.resolve(__dirname, "..", "build", "webpack"),
+    filename: "node-fetch.bundle.js",
+    library: {
+      name: "NodeFetch",
+      type: "commonjs",
+    },
+    clean: true,
+    asyncChunks: false, // This is required so that only one file is created
+  },
+  mode: "production",
+  target: "electron-renderer",
+  optimization: {
+    concatenateModules: true,
+    minimize: true,
+  },
+  externals: {
+    "node:buffer": "commonjs node:buffer",
```

Comment:
Why these specific externals? Could we just use node externals here (meaning everything)?

Reply:
Let me try again, it has been too long since I initially did this so I forget why that wasn't enough.